src/macros.jl

```diff
@@ -225,6 +226,23 @@ If given a `GroupedDataFrame`, `@where` applies transformations by
 group, and returns a fresh `DataFrame` containing the rows
 for which the generated values are all `true`.
 
+Inputs to `@where` can come in two formats: a `block`, in which case each
```

Comment:
"Block" isn't a language keyword, right? Also isn't super easy to understand. Maybe "as a `begin... end` block" and say "line" instead of "argument"?

(Same for other functions.)

Reply:
Fixed all of these.